app/models/subnet/ipv6.rb

```diff
@@ -23,7 +23,13 @@ def validate_ip(ip)
   end
 
   def self.supported_ipam_modes
-    [:eui64, :none]
+    [:eui64, :db, :none]
+  end
```

Comment:
The most common ways to assign an IPv6 address to a host are the following:
* Use SLAAC
* Use DHCPv6

SLAAC actually uses EUI64 to generate an IPv6 address out of the MAC Address, this totally happens on the client side so Foreman is not involved and does not have any knowledge (except for facts) of the address. DHCPv6 can use any kind of address. EUI64 is Foreman specific. Foreman generates the IPv6 address for the client. On the host it can then be set via kickstart or via Puppet. This allows Foreman to set dns aaaa records for this ip. This is good for a dual stacked approach where provisioning still happens completely via IPv4 but the hosts get an additional IPv6 address. Provisioning via IPv6 only needs some more engineering work and at least your EFI support.